src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -190,6 +190,20 @@
     </BoolProperty.Metadata>
   </BoolProperty>
 
+  <BoolProperty Name="AutoGenerateBindingRedirects"
+                DisplayName="Auto-generate binding redirects"
+                Description="Enables binding redirection to be added automatically to the app configuration file to override assembly unification."
```

Comment:
not sure what the best phrasing for this would be @mckennabarlow. How about a more direct description like "Enables automatic addition of binding redirects to the app configuration file to override assembly unification."

Reply:
I agree wording is not the best, I think I ended taking what the documentation describes it as (see helpurl below). @mckennabarlow or @cartermp, feel free to propose.